packages/components/pie-input/src/index.ts

```diff
@@ -154,8 +163,11 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
             inputmode,
             maxlength,
             minlength,
+            min,
+            max,
             name,
             pattern,
+            step,
```

Comment:
Sorry for the delay commenting on this (was just having a nose back through the last weeks worth I have built up!), but if I didn't NIT on `step` not being in alphabetical order like the rest of the attributes, I couldn't live with myself 😆  (`max` as well though too)

(I also can't believe @xander-marjoram let this one through 🤣 )

Reply:
Also, everything else here looks great btw 💯 🚀

Reply:
I did it to hurt you @ashleynolan